tasks/post-to-cmr/index.js

```diff
@@ -44,7 +43,7 @@ function buildOutput(results, granulesObject) {
  * @returns {Promise} returns the promise of an updated event object
  */
 async function postToCMR(event) {
-  // we have to post the meta-xml file of all output granules
+  // we have to post the metadata file for the output granules files
```

Comment:
`the metadata file for the output granules`. I don't think `granules files` is accurate.

Reply:
fa6756a8c